lib/awesome_nested_set/move.rb

```diff
@@ -96,7 +96,7 @@ def case_condition_for_parent
 
         def lock_nodes_between!(left_bound, right_bound)
           # select the rows in the model between a and d, and apply a lock
-          instance_base_class.nested_set_scope.
+          instance_base_class.default_scoped.nested_set_scope.
```

Comment:
Substituting `default_scoped` with `unscoped` makes the specs pass as well.
Looking at the previous implementation it seems unscoped could be used, but the scope was probably inherited, hence the warnings...
https://github.com/collectiveidea/awesome_nested_set/commit/6c5040c4c116cbff2bea69724972cff500f8334f#diff-b1bc4bf228f372cd9fa501fa4d3ade576476fba30f75fbc2ec6de15fb9004444L99

Reply:
This is a tricky one. My typical case where things can get tricky is a default scope that filters out records like
```
default_scope -> { where(deleted: false) }
```

In this case it might seem safer to use `unscoped` and make sure you are locking all records

Reply:
@marcrohloff Good point! I think you are correct. I've created a PR to change this:
https://github.com/collectiveidea/awesome_nested_set/pull/450

Reply:
Ok that PR still needs some work...